src/Server/HTTPHandler.cpp

```diff
@@ -1060,6 +1060,9 @@ void HTTPHandler::handleRequest(HTTPServerRequest & request, HTTPServerResponse
             context->getOpenTelemetrySpanLog());
         thread_trace_context->root_span.kind = OpenTelemetry::SERVER;
         thread_trace_context->root_span.addAttribute("clickhouse.uri", request.getURI());
+        thread_trace_context->root_span.addAttribute("http_referer",  session->getClientInfo().http_referer);
```

Comment:
for these attribute names, I'm suggesting to follow the newest OpenTelemetry standard attribute names for http server spans.

https://opentelemetry.io/docs/specs/semconv/http/http-spans/#http-server-definitions

Reply:
Fixed. Thanks for the suggestion.